app/views/admin/_navbar.html.haml

```diff
@@ -0,0 +1,18 @@
+%nav.navbar.navbar-expand-lg.navbar-dark.bg-dark
+  = link_to "Hello, #{@current_user.name}!", '/admin', class: ".navbar-brand"
+  %button.navbar-toggler{:type => "button", "data-toggle" => "collapse", "data-target" => "#navbarNav", "aria-controls" => "navbarNav", "aria-expanded" => "false", "aria-label" => "Toggle navigation"}
```

Comment:
please use new syntax `type: "button"` and add space at the beginning and end of brackets {}